From 0072eeb6a3c523eff6b00503f708ed09842735c0 Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Tue, 8 Mar 2022 13:29:40 +0100
Subject: [PATCH 0113/1141] arm64: defconfig: enable STM32 I2C support in
 addons fragment

Add compilation as built-in of the I2C_STM32F7 in
fragment-02-defconfig-addons.config for arm64.

Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Change-Id: Idb1de4f4573c76a158c5dfe6bb35dfef7175d104
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/302058
Tested-by: Valentin CARON <valentin.caron@foss.st.com>
Reviewed-by: Valentin CARON <valentin.caron@foss.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -3,6 +3,7 @@ CONFIG_ARCH_STM32=y
 CONFIG_ARM_SCMI_CPUFREQ=y
 CONFIG_ARM_SMC_WATCHDOG=y
 CONFIG_DYNAMIC_DEBUG=y
+CONFIG_I2C_STM32F7=y
 CONFIG_REGULATOR_ARM_SCMI=y
 CONFIG_RTC_DRV_STM32=y
 CONFIG_SERIAL_STM32=y
